feeds/wifi-trunk/hostapd/files/hostapd.sh

```diff
@@ -261,8 +261,6 @@ hostapd_common_add_bss_config() {
 	
 	config_add_string 'owe_transition_bssid:macaddr' 'owe_transition_ssid:string'
 
-	config_add_int beacon_rate
-	config_add_int rssi_reject_assoc_rssi
```

Comment:
this looks odd, I would assume that just not adding the uci would be enough.

Reply:
This was causing problems on ecw5211, as mentioned by @RickSommerville. The problematic patch was introduced in one of the previous commits, and it was promptly followed by another one which was supposed to undo the changes. Apparently, as mentioned by @RickSommerville, the fix was incomplete. I removed those lines in order to prevent **hostapd** from crashing on **ecw5211**.

Reply:
Hi, I don't agree with removing rssi_reject_assoc_rssi.  Why is rssi_reject_assoc_rssi getting removed?